packages/integrations/alpinejs/src/index.ts

```diff
@@ -1,16 +1,83 @@
 import type { AstroIntegration } from 'astro';
+import type { Plugin } from 'vite';
+import { resolve } from 'node:path';
 
-export default function createPlugin(): AstroIntegration {
+interface Options {
+	entrypoint?: string;
```

Comment:
Can we document this option?

Reply:
With JSDoc you mean? FYI the vue integration does not document options with JSDoc so I didn't do it but I can!

Reply:
Yeah with JSDoc, so users can see the description of the options (I think)